boto/pyami/config.py

```diff
@@ -70,6 +68,20 @@ def __init__(self, path=None, fp=None, do_load=True):
                 except IOError:
                     warnings.warn('Unable to load AWS_CREDENTIAL_FILE (%s)' % full_path)
 
+    def __getattr__(self, name):
+        try:
+            impl = self.__dict__['_impl']
+        except KeyError:
+            raise AttributeError(name)
+        return getattr(impl, name)
+
+    def has_option(self, *args, **kwargs):
+        try:
+            impl = self.__dict__['_impl']
```

Comment:
I had a strange pickle-related failure (if I remember correctly) due to missing _impl, but I can't find/reproduce it. It may have been caused by the way I patched the release-version boto with the fixed methods.